doc/arithmetic_parser_user_guide.md

```diff
@@ -11,9 +11,9 @@
       ²   /    <   ≠   ∪    ∨
       ⁻¹  mod  >   ≤   in   ?:
       !   ×    <=  ≥   not  |absolute-value|
-      °   ÷    >=  ∈   and
```

Comment:
Let's just cut those whole list, as it is a duplicate of the parts below (which is why you had to add '//' in two places - bad). I think I originally pasted it in just so I remembered to document all the operators.

Reply:
What do you mean?

Reply:
I think the descriptions for all these operators are repeated in the sections immediately following. So all this list does is make extra maintenance work for us.

Reply:
Modified